source/help/settings/account-settings.md

```diff
@@ -88,14 +88,16 @@ Configure how names are displayed in Mattermost: nickname, username or full name
 Select if the text in the center channel is fixed width and centered, or full width. 
 
 #### Language
-Select what language Mattermost displays in the user interface. Options include: 
+Select what language Mattermost displays in the user interface. Options currently include: 
```

Comment:
Don't think "currently" or "check back later" is useful for end user documentation, this is installed software, not a SaaS service.